client/src/components/DataFiles/DataFilesModals/DataFilesRenameModal.jsx

```diff
@@ -41,7 +41,11 @@ const DataFilesRenameModal = () => {
   });
 
   const onClosed = () => {
-    setStatus(null);
+    /* 
```

Comment:
If the function isn't doing anything, we should use something like the other onClosed functions we have instead of an empty function.

Reply:
DataFilesShowPathModal.jsx uses a DATA_FILES_MODAL_CLOSE dispatch call. I'm assuming this is used to monitor that there was a recently closed modal. I'll be adding that to the onClosed call. I'm open if there are any other suggestions on top of that as well!